shared/libraries/websocket_streaming/src/streaming_server.cpp

```diff
@@ -395,4 +362,126 @@ void StreamingServer::unsubscribeHandler(const std::string& signalId, OutputSign
     }
 }
 
+void StreamingServer::addSignals(const ListPtr<ISignal>& signals)
+{
+    for (auto& [_, client] : clients)
+    {
+        auto writer = client.first;
+        auto& outputSignals = client.second;
+
+        publishSignalsToClient(writer, signals, outputSignals);
+    }
+}
+
+void StreamingServer::removeComponentSignals(const StringPtr& componentId)
+{
+    auto removedComponentId = componentId.toStdString();
+
+    for (auto& [_, client] : clients)
+    {
+        auto writer = client.first;
+        auto& outputSignals = client.second;
+
+        std::vector<std::string> signalsToRemove;
+
+        for (const auto& [signalId, outputSignal] : outputSignals)
+        {
+            // removed component is a signal, or signal is a descendant of removed component
+            if (signalId == removedComponentId || IdsParser::isNestedComponentId(removedComponentId, signalId))
+            {
+                signalsToRemove.push_back(signalId);
+                unsubscribeHandler(signalId, outputSignal);
+            }
+        }
+        for (const auto& signalId : signalsToRemove)
+        {
+            outputSignals.erase(signalId);
+        }
+
+        writeSignalsUnavailable(writer, signalsToRemove);
+    }
+}
+
+void StreamingServer::updateOutputValueSignal(OutputSignalBasePtr& outputSignal,
+                                              SignalMap& outputSignals,
+                                              const StreamWriterPtr& writer)
+{
+    auto placeholderSignal = std::dynamic_pointer_cast<OutputNullSignal>(outputSignal);
+
+    if (placeholderSignal)
+    {
+        auto daqSignal = outputSignal->getDaqSignal();
+        auto signalId = daqSignal.getGlobalId().toStdString();
+
+        LOG_I("Parameters of unsupported signal {} has been changed, check if it is supported now ...", daqSignal.getGlobalId());
+        try
+        {
+            addToOutputSignals(daqSignal, outputSignals, writer);
+            outputSignal = outputSignals.at(signalId);
+
+            if (placeholderSignal->isSubscribed())
+                outputSignal->setSubscribed(true);
+        }
+        catch (const DaqException& e)
+        {
+            LOG_W("Failed to create an output LT streaming signal for {}: {}", daqSignal.getGlobalId(), e.what());
+        }
```

Comment:
This warning likely appears quite often, as only a select amount of signal types are supported. It'd be great if this could be a bit more descriptive and have a reason as to why the signal isn't added. Eg. "Failed to create LT streaming output signal. Signals with that have an explicit rule domain signal are not supported in LT streaming."

Similarly, warning logs below should be a bit more descriptive.